src/ui/mappixmapitem.cpp

```diff
@@ -95,6 +96,7 @@ void MapPixmapItem::shift(int xDelta, int yDelta, bool fromScriptCall) {
     }
 
     if (!fromScriptCall && map->layout->blockdata != oldMetatiles) {
+        Scripting::cb_MapShifted(xDelta, yDelta);
```

Comment:
Same thing here--call the script hook should probably be called after the map has actually been shifted.

Reply:
The map shift occurs in the block above, so it already is.

Reply:
Oh unless you mean it should also occur after the change to the edit history right below it

Reply:
I had misread the code, but yeah, I still think it should go after the edit history line.